1. CWE Base Mod/Cold War Era Mod (CWE)/common/goods/00_goods.txt

```diff
@@ -33,7 +33,7 @@ advanced_ammunition = { #Covers smart weapons
 
 refined_uranium = { #For nuclear weapons, produced by nuclear power plants
 	texture = "gfx/interface/icons/goods_icons/refined_uranium.dds"
-	cost = 20
+	cost = 50
```

Comment:
Base prices should be kept uniform at 10 for raw resource, 20 for intermediate and 40 for completed products. This should not be an issue as actual prices will be determined by market forces anyway.